Sources/Core/ObjectiveCIR.swift

```diff
@@ -379,11 +432,21 @@ public struct ObjCIR {
                     }).joined(separator: "\n"),
                     "@end",
                 ].map { $0.trimmingCharacters(in: CharacterSet.whitespaces) }.filter { $0 != "" }
-            case let .category(className: className, categoryName: categoryName, methods: methods, properties: properties):
+            case let .category(className: className, categoryName: categoryName, methods: methods, properties: properties, variables: variables):
                 // Only render anonymous categories in the implementation
                 guard categoryName == nil else { return [] }
+                let variableDeclarations: String
+                if !variables.isEmpty {
+                    let vars: [String] = variables.map { (param, typeName) -> String in
+                        "\t\(typeName) _\(Languages.objectiveC.snakeCaseToPropertyName(param));"
```

Comment:
We have a custom operator that you can utilize here to do the tabbing for you. 

Do a quick grep for `git grep "\-->"` and you'll other instances

Reply:
done.